src/util/get-github-client-config.ts

```diff
@@ -27,13 +29,15 @@ export async function getGitHubApiUrl(jiraHost: string, gitHubAppId: number) {
 		: GITHUB_CLOUD_API_BASEURL;
 }
 
-const getGitHubClientConfigFromAppId = async (gitHubAppId: number | undefined, jiraHost?: string): Promise<GitHubClientConfig> => {
+export const getGitHubClientConfigFromAppId = async (gitHubAppId: number | undefined, jiraHost?: string): Promise<GitHubClientConfig> => {
 	const gitHubServerApp = gitHubAppId && await GitHubServerApp.getForGitHubServerAppId(gitHubAppId);
 	if (gitHubServerApp) {
 		return	{
 			hostname: gitHubServerApp.gitHubBaseUrl,
-			baseUrl: gitHubServerApp.gitHubBaseUrl,
+			baseUrl: `${gitHubServerApp.gitHubBaseUrl}/api/v3`,
```

Comment:
**BLOCKING**
This is super weird, `gitHubServerApp.gitHubBaseUrl` !== `gitHubClientConfig.baseUrl`

Reply:
This has been reverted